oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/util/TimingDocumentStoreWrapper.java

```diff
@@ -443,6 +444,23 @@ public int getNodeNameLimit() {
         }
     }
 
+    /**
+     * Return the @{@link ThrottlingMetrics} for the underlying document store
+     *
+     * @return throttling metric for document store
+     */
+    @Override
+    public ThrottlingMetrics throttlingMetrics() {
+        try {
+            long start = now();
+            final ThrottlingMetrics result = base.throttlingMetrics();
+            updateAndLogTimes("getNodeNameLimit", start, 0, 0);
```

Comment:
Wrong operation name.
```suggestion
            updateAndLogTimes("throttlingMetrics", start, 0, 0);
```